setup/rpi_raspbian.md

```diff
@@ -0,0 +1,86 @@
+# Using Raspbian on the RPi instead of Ubuntu
+
+**These instructions are up-to-date as of 8/17/2020, but will not be regularly updated**
+
+## 1 Intalling Raspbian
+
+To install Raspbian, we recommend following the "Getting started with Raspberry Pi" tutorial below. 
+
+**NOTE: These instructions were tested and verified using Raspbian Stretch. Newer versions of Raspbian (such as Buster) may cause complications with these instructions. We therefore recommend installing Raspbian Stretch instead of the latest/default version of Raspbian, which the tutorial below will recommend. Do not flash NOOBS onto your card; instead download the Raspbian Streth image below and skip the NOOBS step in the tutorial.** 
+
+In addition, we have created a pre-built image with both Raspbian Stretch and ROS already installed. You can download this image as a last resort if you can’t get Raspbian and ROS installed and working on your own, but please note that you will miss out on some the learning opportunities that come with debugging a software installation by doing so. 
+
+- [Tutorial](https://projects.raspberrypi.org/en/projects/raspberry-pi-getting-started) for installing the latest version of Raspbian on a new Raspberry Pi
+- Direct downloadable Raspbian Stretch [image](https://downloads.raspberrypi.org/raspbian/images/raspbian-2019-04-09/2019-04-08-raspbian-stretch.zip)
+- Custom JPL [image](https://drive.google.com/drive/folders/1RbYWDRthpcktqkPEHb7qVqDiy27EiAc1) with both Raspbian Stretch and ROS pre-built
+
+**NOTE: If you install the Custom JPL image with Raspbian Stretch and ROS pre-built, skip to step 2.4.**
+
+Once you finish the above tutorial and install Raspbian Stretch, you should be able to boot your Raspberry Pi and see a desktop!
+
+
+
+
+
+## 2 Installing ROS Kinetic on Raspbian
+
+Use these instructions if you followed subsection 2.1.2 above.
+
+If you do not download the pre-built Raspbian Stretch / ROS image, you will need to build ROS yourself.
+
+First, make sure you update all the packages on your pi:
+
+```
+sudo apt-get update
+sudo apt-get upgrade
+```
+
+(The above commands may take a few minutes to run.)
+
+Next, we will install ROS (and specifically, the version called ’Kinetic’).
+
+**NOTE: Follow the ROS installation directions in the link below CAREFULLY! Do not ignore warnings, as they will lead to issues later in the installation. Some known ”gotchas” to note before you get started are:**
+
+- Make sure you install the Kinetic ”desktop full” version of ROS, not the desktop ros-comm, or other versions.
```

Comment:
This isn't necessary. install ros-kinetic-base will work just fine. No need to install RViz and other stuff

Reply:
Honestly I think I'll just keep this as is for now. It's the old raspbian instructions. Don't really feel like touching them (beyond moving them) in this PR.